### DISCUSSION TITLE
MediaPlayerElement is already supported in winappsdk 1.2, but not yet in uno 4.6

### DISCUSSION THREAD

**Discussion by Daoting** [unoplatform/uno#10831] [Q&A]
MediaPlayerElement is already supported in winappsdk 1.2, but not yet in uno 4.6.

https://github.com/unoplatform/uno/discussions/9318

**Comment by Daoting** [comment]
Same Discussions:
https://github.com/unoplatform/uno/discussions/10852
https://github.com/unoplatform/uno/discussions/9318

@jeromelaban

**Reply by jeromelaban** [reply]
[`MediaPlayerElement`](https://github.com/unoplatform/uno/blob/441df8336e56656813a0ec7834b5c554f392f7d3/src/Uno.UI/UI/Xaml/Controls/MediaPlayer/MediaPlayerElement.cs) is available for mobile targets. Which platform are you trying to use this control with?

**Reply by Daoting** [reply]
I tried again and even the latest version 4.7.0-dev.858 is not available on mobile.
My reference is the Uno.WinUI package. 
Please check for it.

**Reply by jeromelaban** [reply]
Indeed, [it's disabled for WinUI](https://github.com/unoplatform/uno/blob/441df8336e56656813a0ec7834b5c554f392f7d3/src/Uno.UI/UI/Xaml/Controls/MediaPlayer/MediaPlayerElement.cs#L1), though it should have been. Thanks for looking into it.
